### ISSUE TITLE
MDG take over meteor-apollo-accounts

### ISSUE THREAD

**Issue by acomito** [meteor/meteor-feature-requests#323]
### MDG take over meteor-apollo-accounts

I love using meteor to bootstrap my back-end and as a build system. However, I like to use apollo/graphql/react and keep my client-side code as "meteor free" as possible (just in case I ever need to move it). This [meteor-apollo-accounts](https://github.com/orionsoft/meteor-apollo-accounts)  a great package. It's like 90% there. The maintainer has been great but needs some help.

If you use graphql with meteor-apollo-accounts then you're not "locked in" on the client side with a ton of Meteor.user() across 300 files.  This setup is amazing... free account system, great build system, graphql, out-of-the-box mongo and you can have it hosted on Galaxy within 2 minutes and virtually zero config and it's more a "node app" than a "meteor app". The issue is meteor-apollo-accounts is a little rough around the edges and I think it (or similar) should be officially supported by MDG. It's the vital package that really closes the loop between apollo and meteor in my opinion.

**Comment by aadamsx** [comment]
100% agree. I’m looking at this package now and would love to know this in looped into the official meteor ecosystem.

**Comment by acomito** [comment]
I don't think it would be too much overhead for them-- of course, everything is easy when somebody else is doing the work-- but it's in good shape it just needs a little more of an active maintainer for PRs and stuff. It just seems like such a vital piece of the puzzle that MDG should have the wheel.

**Comment by StorytellerCZ** [comment]
Closing as duplicate of #290
